app/backend/Dockerfile

```diff
@@ -10,7 +10,7 @@ ENV PYTHONBUFFERED=1
 RUN set -ex; \
   apt-get update && \
   apt-get upgrade -y && \
-  apt-get install -y build-essential curl && \
+  apt-get install -y build-essential curl zlib1g-dev && \
```

Comment:
MacOS-specific @BowenXU0126; when running `make devbackend`, without this library it complains about missing header `<zlib.h>` for him.